Add support for basic auth to Kafka schema registry (#25456)
trinodb/trino: docs/src/main/sphinx/connector/kafka.md

  - Comma-separated list of URL addresses for the Confluent schema registry.
    For example, `http://schema-registry-1.example.org:8081,http://schema-registry-2.example.org:8081`
  -
* - `kafka.confluent-schema-registry-auth-type`
  - The type of authentication implemented on Confluent schema registry.
    Allowed values are: `BASIC_AUTH` and `NONE`

>     * `NONE` - no authentication on schema registry is implemented

Reviewer: ```suggestion
    * `NONE` - No authentication is configured for the Schema Registry
```
